version/command.go

```diff
@@ -22,5 +23,7 @@ func GetVersion() string {
 
 // CMD
 func printVersion(cmd *cobra.Command, args []string) {
-	fmt.Println(GetVersion())
+	fmt.Println("cosmos-sdk", GetVersion())
+	fmt.Println("Built with Go", runtime.Version())
+	fmt.Println("vendor/ directory's hash:", VendorDirHash)
```

Comment:
Could we also include the exact Git commit (maybe also timestamp and message)?

Reply:
Isn't the git commit enough?

Reply:
Haha probably.